lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -803,6 +868,12 @@ test.serial('migrateGranuleAndFilesViaTransaction processes granule with no file
   });
 });
 
+// FUTURE
+test.skip('migrateGranuleAndFilesViaTransaction removes previously migrated files if a granule is re-migrated with an undefined files key', async (t) => {});
```

Comment:
I think instead of (or I'm fine with *in addition to*) putting these here we should make this update an AC in one of the related tickets.   Stub tests are easily forgotten.

Reply:
Made a note in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3077. I sorta like leaving the test stubs because they're explicit. Noting them in the ticket might be sufficient to not forget about them? The AC in that ticket looks like it already covers them.